cms/templates/cms/home_page.html

```diff
@@ -75,7 +75,46 @@
         </div>
       </div>
 
-            <section class="row current-programs-section">
+      <div class="row how-it-works-section">
```

Comment:
We have not been consistent unfortunately. I basically copied the benefits section that was not using <section>. I just didn't want to break anything before the release.